packages/perseus/src/components/math-input.tsx

```diff
@@ -208,15 +223,12 @@ class MathInput extends React.Component<Props, State> {
         return this.state.focused;
     };
 
-    insert: (arg1: any) => void = (value) => {
+    insert: (keyPressed: Key) => void = (keyPressed: Key) => {
```

Comment:
**nit**: If we aren't passing this function off to some other context, we could just make this an instance function and simplify the syntax a bunch here. 

```suggestion
    insert(keyPressed: Key): void {
```